content/get-started/quickstart/git-and-github-learning-resources.md

```diff
@@ -18,7 +18,7 @@ shortTitle: Learning resources
 ---
 ## Using Git
 
-Familiarize yourself with Git by visiting the [official Git project site](https://git-scm.com) and reading the [ProGit book](http://git-scm.com/book). You can review the [Git command list](https://git-scm.com/docs) or [Git command lookup reference](http://gitref.org) while using the [Try Git](https://try.github.com) simulator.
+Familiarize yourself with Git by visiting the [official Git project site](https://git-scm.com) and reading the [ProGit book](http://git-scm.com/book). You can also review the [Git command list](https://git-scm.com/docs) or our [Git command lookup reference](https://git.github.io/git-reference/).
```

Comment:
Thank you for removing the outdated link a git simulator and for tracking down the Git command reference that's backed by https://github.com/git/git-reference. 💖 

It looks as if the Git command reference is no longer actively maintained, so I suggest we remove this reference too.

```suggestion
Familiarize yourself with Git by visiting the [official Git project site](https://git-scm.com) and reading the [ProGit book](http://git-scm.com/book). You can also review the [Git command list](https://git-scm.com/docs).
```

Reply:
Well, I agree it does not look actively maintained, but it did help me at first when I was learning Git. So it's you call :)

Reply:
It's good to know you found this resource useful, even though it's not actively maintained. I guess the core functionality of Git doesn't change much.